lib/hocon/impl/config_impl.rb

```diff
@@ -191,15 +191,44 @@ def self.trace(message, indent_level = 0)
     $stderr.puts(message)
   end
 
+  def self.empty_object_from_origin(origin)
+    # we want null origin to go to SimpleConfigObject.empty() to get the
+    # origin "empty config" rather than "hardcoded value"
```

Comment:
I read this comment 3 times, couldn't figure out what it meant, and was about to ask you to re-word it ... and then I realized that it's copied from upstream.  :8ball: